paimon-web-flink/src/main/java/org/apache/paimon/web/flink/task/FlinkTask.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.paimon.web.flink.task;
+
+import org.apache.paimon.web.common.data.vo.SubmitResult;
+import org.apache.paimon.web.task.SubmitTask;
+
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.TableEnvironment;
+import org.apache.flink.table.api.TableResult;
+import org.apache.flink.types.Row;
+import org.apache.flink.util.CloseableIterator;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FlinkTask implements SubmitTask {
+    private static final StreamExecutionEnvironment env =
+            StreamExecutionEnvironment.getExecutionEnvironment();
+    private static final TableEnvironment tEnv = TableEnvironment.create(new Configuration());
+    private static final String nullColumn = "";
+
+    public void executeDDL(String statement) throws Exception {
+        tEnv.executeSql(statement);
+    }
+
+    @Override
+    public SubmitResult execute(String statement) throws Exception {
+        TableResult tableResult = tEnv.executeSql(statement);
```

Comment:
Can we just use restapi here? You can take a look to https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/table/sql-gateway/overview/